application/microservices/entando-hub-catalog-ms/src/test/java/com/entando/hub/catalog/integration/CatalogFlowIT.java

```diff
@@ -236,7 +239,7 @@ void shouldReturnForbiddenWhenCreateCatalogWithNoAdminRole() throws Exception {
     void shouldReturnUnauthorizedWhenCreateCatalogWithNoRole() throws Exception {
```

Comment:
Please rename the method name with something like shouldReturnForbiddenWhenCreateCatalogWithNoRole

Reply:
Done!